db1/toCSV.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const fsPromise = require('fs').promises;
+// eslint-disable-next-line object-curly-newline
+const { usersTable, Review, ReviewFeedbacks, productsTable } = require('./fakerData');
+
+const chunk = 100000;
+const maxRecords = 1000000;
+
+const usersCSV = () => {
+  const dato = new Date();
+  console.log('userCSV:', dato.getMinutes(), dato.getSeconds());
+  fs.writeFile('userTable.csv', usersTable());
+  const ddato = new Date();
+  console.log('userCSV:', ddato.getMinutes(), ddato.getSeconds());
```

Comment:
Really cleaver idea for timing your loading times I really like it, I was using my phone, but I am going to implement something like this.